src/panels/lovelace/editor/config-elements/hui-entity-filter-card-editor.ts

```diff
@@ -0,0 +1,403 @@
+                .lovelace=${this.lovelace}
+                @config-changed=${this._handleCardConfigChanged}
+                @GUImode-changed=${this._cardGUIModeChanged}
+              ></hui-element-editor>
+            `
+          : html`
+              <hui-card-picker
+                .hass=${this.hass}
+                .lovelace=${this.lovelace}
+                @config-changed="${this._pickCard}"
+              ></hui-card-picker>
+            `}
+      </div>
+    `;
+  }
+
+  private _handleSwitchTab(ev) {
+    this._selectedTab = parseInt(ev.detail.selected, 10);
+  }
+
+  private _showEmptyToggle(): void {
+    if (!this._config || !this.hass) {
+      return;
+    }
+    this._config = {
+      ...this._config,
+      show_empty: this._config.show_empty === false,
+    };
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _entitiesChanged(ev: EntitiesEditorEvent): void {
+    if (!this._config || !this.hass) {
+      return;
+    }
+    if (!ev.detail || !ev.detail.entities) {
+      return;
+    }
+
+    this._config = {
+      ...this._config,
+      entities: ev.detail.entities as EntityFilterEntityConfig[],
+    };
+    this._configEntities = processEditorEntities(this._config.entities);
+
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _stateDeleted(ev: Event): void {
+    const target = ev.target! as any;
+    if (target.value === "" || !this._config) {
+      return;
+    }
+    const state_filter = [...this._config.state_filter];
+    state_filter.splice(target.index, 1);
+
+    this._config = { ...this._config, state_filter };
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _stateAdded(ev: Event): void {
+    const target = ev.target! as any;
+    if (target.value === "" || !this._config) {
+      return;
+    }
+    const state_filter = [...this._config.state_filter];
+    state_filter.push(target.value);
+
+    this._config = { ...this._config, state_filter };
+    target.value = "";
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _stateChanged(ev: Event): void {
+    const target = ev.target! as any;
+    if (target.value === "" || !this._config) {
+      return;
+    }
+    const state_filter = [...this._config.state_filter];
+    state_filter[target.index] = target.value;
+
+    this._config = { ...this._config, state_filter };
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _cardGUIModeChanged(ev: HASSDomEvent<GUIModeChangedEvent>): void {
+    ev.stopPropagation();
+    this._cardGUImode = ev.detail.guiMode;
+    this._cardGUIModeAvailable = ev.detail.guiModeAvailable;
+  }
+
+  private _toggleCardMode(): void {
+    this._cardEditorEl?.toggleMode();
+  }
+
+  private _getCardConfig(): LovelaceCardConfig {
+    const cardConfig = { ...this._config!.card } as LovelaceCardConfig;
+    cardConfig.entities = ["auto"];
```

Comment:
This is a magic value to make the entity pickers not render.
`card.entities` is removed before saving the configuration.

I'm not particularly fond of this solution, so if anyone have a better idea I'm all ears.

Reply:
Yeah we should do that different... I'll think a bit

Reply:
We could add a property to `hui-card-element-editor` that would pass it to `configElement`?